src/content/configuration/entry-context.mdx

```diff
@@ -175,6 +176,19 @@ module.exports = {
 
 For example: you can use dynamic entries to get the actual entries from an external source (remote server, file system content or database):
 
+### Multiple Entry Points in Different Folders
+In some scenarios, you might want to organize your entry points into different folders for better structure and maintainability. Here's how you can define multiple entry points located in different folders:
+```
+
+module.exports = {
+  //...
+  entry: {
+    '/scriptFolder/index.min': '../Static/index.js',
```

Comment:
maybe simplify the entry to local relative paths? Makes it easier to understand. 

`./js/one.js`and `./js/two.js`

Reply:
yes, thats better.